### ISSUE TITLE
All layers skipped

### ISSUE THREAD

**Issue by besherh** [CalculatedContent/WeightWatcher#15]
I am trying to test on a simple model inspired from here:
https://keras.io/examples/cifar10_cnn/

The result shows after applying ww, all layers are skipped. Here is the model summary: 
```
Model: "sequential_6"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
conv2d_24 (Conv2D)           (None, 32, 32, 32)        896       
_________________________________________________________________
activation_36 (Activation)   (None, 32, 32, 32)        0         
_________________________________________________________________
conv2d_25 (Conv2D)           (None, 30, 30, 32)        9248      
_________________________________________________________________
activation_37 (Activation)   (None, 30, 30, 32)        0         
_________________________________________________________________
max_pooling2d_12 (MaxPooling (None, 15, 15, 32)        0         
_________________________________________________________________
dropout_18 (Dropout)         (None, 15, 15, 32)        0         
_________________________________________________________________
conv2d_26 (Conv2D)           (None, 15, 15, 64)        18496     
_________________________________________________________________
activation_38 (Activation)   (None, 15, 15, 64)        0         
_________________________________________________________________
conv2d_27 (Conv2D)           (None, 13, 13, 64)        36928     
_________________________________________________________________
activation_39 (Activation)   (None, 13, 13, 64)        0         
_________________________________________________________________
max_pooling2d_13 (MaxPooling (None, 6, 6, 64)          0         
_________________________________________________________________
dropout_19 (Dropout)         (None, 6, 6, 64)          0         
_________________________________________________________________
flatten_6 (Flatten)          (None, 2304)              0         
_________________________________________________________________
dense_12 (Dense)             (None, 512)               1180160   
_________________________________________________________________
activation_40 (Activation)   (None, 512)               0         
_________________________________________________________________
dropout_20 (Dropout)         (None, 512)               0         
_________________________________________________________________
dense_13 (Dense)             (None, 10)                5130      
_________________________________________________________________
activation_41 (Activation)   (None, 10)                0         
=================================================================
Total params: 1,250,858
Trainable params: 1,250,858
Non-trainable params: 0
```

WW log:

'''
{0: {'id': 0,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.convolutional.Conv2D at 0x7f812430fd68>},
 1: {'id': 1,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Activation at 0x7f812430b3c8>},
 2: {'id': 2,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.convolutional.Conv2D at 0x7f812430b630>},
 3: {'id': 3,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Activation at 0x7f81242ff710>},
 4: {'id': 4,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.pooling.MaxPooling2D at 0x7f81242ff898>},
 5: {'id': 5,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Dropout at 0x7f8123eb2898>},
 6: {'id': 6,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.convolutional.Conv2D at 0x7f8123eb2cc0>},
 7: {'id': 7,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Activation at 0x7f8123e823c8>},
 8: {'id': 8,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.convolutional.Conv2D at 0x7f8123e82240>},
 9: {'id': 9,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Activation at 0x7f8123e8b7f0>},
 10: {'id': 10,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.pooling.MaxPooling2D at 0x7f8123e8b898>},
 11: {'id': 11,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Dropout at 0x7f8123ea0dd8>},
 12: {'id': 12,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Flatten at 0x7f8123ea0c50>},
 13: {'id': 13,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Dense at 0x7f8123ea0c18>},
 14: {'id': 14,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Activation at 0x7f8123def9b0>},
 15: {'id': 15,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Dropout at 0x7f8123def9e8>},
 16: {'id': 16,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Dense at 0x7f8123defcc0>},
 17: {'id': 17,
  'message': 'Skipping (Layer not supported)',
  'type': <tensorflow.python.keras.layers.core.Activation at 0x7f8123da4198>}}
'''

**Comment by charlesmartin14** [comment]
Right
The layers are very small and you may not get a good power law fit

To treat this system, set MIN_SIZE=0

**Comment by besherh** [comment]
It's still the same, all are skipped: 
results = watcher.analyze(compute_alphas=True,min_size=6)

**Comment by charlesmartin14** [comment]
Thats odd...maybe there is an issue with this version of keras
I will take a look but in a few days

In the mean time, let me suggest trying one of the pytorch pretrained models
thanks

**Comment by charlesmartin14** [comment]
BTW, are you using the pypi version (0.1.2) or deploying from a branch or source ?

**Comment by besherh** [comment]
I installed it on colab using pip install.
